src/containers/stage.jsx

```diff
@@ -6,6 +6,7 @@ import VM from 'scratch-vm';
 import {connect} from 'react-redux';
 
 import {getEventXY} from '../lib/touch-utils';
+import {VideoProvider} from '../lib/camera';
```

Comment:
I'd rather `import VideoProvider from '../lib/video-provider' -- let it require whatever it wants from `lib/camera`